docs/en/latest/plugins/openid-connect.md

```diff
@@ -51,6 +51,7 @@ The `openid-connect` Plugin provides authentication and introspection capability
 | public_key                           | string  | False    |                       |              | Public key to verify the token.                                                                                    |
 | use_jwks                             | boolean | False    |                       |              | When set to true, uses the JWKS endpoint of the identity server to verify the token.                               |
 | token_signing_alg_values_expected    | string  | False    |                       |              | Algorithm used for signing the authentication token.                                                               |
+| use_pkce                             | boolean | False    |                       |              | when set to true the "Proof Key for Code Exchange" as defined in RFC 7636 will be used.   |
```

Comment:
We should add a `false` default value?

Reply:
yes, a false default value is added in the plugin's source code.

Reply:
Err, I mean we need to doc the default value in the table.